### ISSUE TITLE
Feature: Chat bubble

### ISSUE THREAD

**Issue by shinokada** [themesberg/flowbite-svelte#1183]
## Chat bubble

- [ ] [Default chat bubble](https://flowbite.com/docs/components/chat-bubble/#default-chat-bubble)
- [ ] [Voice note message](https://flowbite.com/docs/components/chat-bubble/#voice-note-message)
- [ ] [File attachment](https://flowbite.com/docs/components/chat-bubble/#file-attachment)
- [ ] [Image attachment](https://flowbite.com/docs/components/chat-bubble/#image-attachment)
- [ ] [Image gallery](https://flowbite.com/docs/components/chat-bubble/#image-gallery)
- [ ] [URL preview sharing](https://flowbite.com/docs/components/chat-bubble/#url-preview-sharing)
- [ ] [Outline chat bubble](https://flowbite.com/docs/components/chat-bubble/#outline-chat-bubble)
- [ ] [Outline voice note](https://flowbite.com/docs/components/chat-bubble/#outline-voice-note)
- [ ] [Outline file attachment](https://flowbite.com/docs/components/chat-bubble/#outline-file-attachment)
- [ ] [Outline image attachment](https://flowbite.com/docs/components/chat-bubble/#outline-image-attachment)
- [ ] [Outline image gallery](https://flowbite.com/docs/components/chat-bubble/#outline-image-gallery)
- [ ] [Outline URL preview sharing](https://flowbite.com/docs/components/chat-bubble/#outline-url-preview-sharing)
- [ ] [Clean chat bubble](https://flowbite.com/docs/components/chat-bubble/#clean-chat-bubble)
- [ ] [Clean voice note](https://flowbite.com/docs/components/chat-bubble/#clean-voice-note)
- [ ] [Clean file attachment](https://flowbite.com/docs/components/chat-bubble/#clean-file-attachment)
- [ ] [Clean image attachment](https://flowbite.com/docs/components/chat-bubble/#clean-image-attachment)
- [ ] [Clean image gallery](https://flowbite.com/docs/components/chat-bubble/#clean-image-gallery)
- [ ] [Clean URL preview sharing](https://flowbite.com/docs/components/chat-bubble/#clean-url-preview-sharing)

**Comment by NielsKrijnen** [comment]
Hey @shinokada, I'm new to open source and I think I can handle this. Can I work on the Chat bubbles?

**Comment by shinokada** [comment]
Yes, you can try it. Why don't you start the default one and create a PR and let's see.
